prow/git/localgit/localgit.go

```diff
@@ -152,3 +152,21 @@ func (lg *LocalGit) RevParse(org, repo, commitlike string) (string, error) {
 	rdir := filepath.Join(lg.Dir, org, repo)
 	return runCmdOutput(lg.Git, rdir, "rev-parse", commitlike)
 }
+
+// MergeCommitsExistBetween does 'git log <target>..<head> --merged'.
+func (lg *LocalGit) MergeCommitsExistBetween(org, repo, target, head string) (string, error) {
```

Comment:
With this your test is validating the `LocalGit` implementation but not the actual git client -- can we use the git client to run the check?

Reply:
Absolutely, and `(lg *LocalGit) MergeCommitsExistBetween()` is removed.